### PR TITLE
Add CapRover icon

### PR DESCRIPTION
**Pull request by CodeShakingSheep**
image

**Issue:** closes none

**Popularity metric:**

It has ~12.6k stars on GitHub. See https://github.com/caprover/caprover

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

<del>I didn't find any svg for this image. So, I used the png from the website's header as base and vectorized it using [SvgTrace](https://svgtrace.com/png-to-svg). Afterwards, I corrected some imperfections in Inkscape.</del>

Icon from GitHub source.

### CODE REVIEW

**Comment by uncenter** [comment]
Did you find https://github.com/caprover/caprover-website/blob/master/graphics/logo.svg?

**Comment by CodeShakingSheep** [comment]
> Did you find https://github.com/caprover/caprover-website/blob/master/graphics/logo.svg?

No, I didn't :smile: Thanks, I just updated the icon and source.

**Comment by uncenter** [comment]
> > Did you find [caprover/caprover-website@`master`/graphics/logo.svg](https://github.com/caprover/caprover-website/blob/master/graphics/logo.svg?rgh-link-date=2024-06-21T03%3A17%3A00Z)?
> 
> No, I didn't 😄 Thanks, I just updated the icon and source.

In the future if there is a GitHub organization you can do `path:*.svg` in the search bar to check for all SVGs in all organization repositories: https://github.com/search?q=org%3Acaprover%20path%3A*.svg&type=code. That's how I found that one!

File: _data/simple-icons.json [hunk 1]
```diff
@@ -2766,6 +2766,11 @@
             "hex": "119EFF",
             "source": "https://github.com/ionic-team/ionicons-site/blob/b0c97018d737b763301154231b34e1b882c0c84d/docs/ionicons/svg/logo-capacitor.svg"
         },
+        {
+            "title": "CapRover",
+            "hex": "EF5C26",
```
Comment (uncenter) [inline _data/simple-icons.json:2771]: I'm not seeing this hex code in the (updated) source icon. I'd maybe go with `#ed5b26`, the color of the largest path in the center of the circle?

File: icons/caprover.svg
Comment (uncenter) [inline icons/caprover.svg:1]: Icon looks good!

**Review by uncenter — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (CodeShakingSheep) [reply _data/simple-icons.json]: Yes, you're right. Just updated the hex code.

**Review by uncenter — APPROVED** [review]

**Pull request merged**